front/src/applications/operationalStudies/utils.ts

```diff
@@ -246,3 +247,5 @@ export const formatSpaceTimeData = (
     trainName: trainName || 'Train name not found',
   };
 };
+export const nameIsInvalid = (name?: string | null) =>
+  (name && name.length > SMALL_INPUT_MAX_LENGTH) || !name;
```

Comment:
You can change the condition like this, it can be read like this: "the name is invalid if it is undefined (or empty string) OR if its length is bigger than the max length (implicitely, the name is not undefined)
```suggestion
  !name || name.length > SMALL_INPUT_MAX_LENGTH;
```

Reply:
Done